core/services/workflows/monitoring.go

```diff
@@ -13,10 +13,18 @@ import (
 )
 
 var registerTriggerFailureCounter metric.Int64Counter
+var triggerWorkflowStarterErrorCounter metric.Int64Counter
 var workflowsRunningGauge metric.Int64Gauge
 var capabilityInvocationCounter metric.Int64Counter
+var capabilityFailureCounter metric.Int64Counter
+var workflowRegisteredCounter metric.Int64Counter
+var workflowUnregisteredCounter metric.Int64Counter
+var workflowExecutionFinishedCounter metric.Int64Counter
 var workflowExecutionLatencyGauge metric.Int64Gauge // ms
 var workflowStepErrorCounter metric.Int64Counter
+var workflowStepStartedCounter metric.Int64Counter
+var workflowStepFinishedCounter metric.Int64Counter
+var workflowInitializationCounter metric.Int64Counter
```

Comment:
Similar comment to https://github.com/smartcontractkit/chainlink/pull/15238/files#r1846926281 above - if we have workflows running by status, does this tell us additional information?